pkg/graph/resolvers/system_intake_grb_reviewer.go

```diff
@@ -240,6 +240,7 @@ func StartGRBReview(
 		}
 
 		intake.GRBReviewStartedAt = helpers.PointerTo(time.Now())
+		intake.GrbReviewAsyncManualEndDate = nil
```

Comment:
Is this just to be sure that it hasn't been set to ended once already? ( I think that makes sense)

Reply:
As with above, this is in accordance with AC spec.